Makefile

```diff
@@ -0,0 +1,39 @@
+ # Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+DOCKER=docker
+BINARY_NAME=adapter
+TARGET=target
+BINARY_LINUX=$(TARGET)/$(BINARY_NAME)_linux
+BINARY_DARWIN=$(TARGET)/$(BINARY_NAME)_darwin
+BINARY_WINDOWS=$(TARGET)/$(BINARY_NAME)_windows.EXE
+
+all: build test
+build: 
+	$(GOBUILD) -o $(BINARY_NAME) -v
+test: 
+	$(GOTEST) -v ./...
+clean: 
+	$(GOCLEAN)
+	rm -f $(BINARY_NAME)
+	rm -rf $(TARGET)
+run:
+	$(GOBUILD) -o $(BINARY_NAME) -v ./...
+	./$(BINARY_NAME)
+deps:
+	$(GOGET) github.com/wavefrontHQ/prometheus-storage-adapter    
```

Comment:
Let's use lowercase `wavefronthq` throughout (Makefile, README and go files) for consistency as we use lowercase for all the other go pkgs under wavefrontHQ.

Reply:
I don't think that's possible, since that's what it's named in github. If you go a "go get", you're going to get names ending with HQ.

Reply:
`go get github.com/wavefronthq/prometheus-storage-adapter` works.

Reply:
You are absolutely right. My bad. Fixed.